src/VisualStudio/Core/Def/ValueTracking/ValueTrackedTreeItemViewModel.cs

```diff
@@ -78,13 +78,24 @@ public override bool IsNodeExpanded
             }
         }
 
+        internal static ValueTask<TreeItemViewModel> CreateAsync(
```

Comment:
Doesn't seem like the overload is worth it since it's only called once. The call site can easily pass empty array.